src/date-range-picker/date-range-picker-embedded.tsx

```diff
@@ -0,0 +1,153 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import React from 'react';
+import { DateRangePickerProps } from './interfaces';
+import Calendar, { DayIndex } from './calendar';
+import InternalBox from '../box/internal';
+import SpaceBetween from '../space-between/index.js';
+
+import styles from './styles.css.js';
+import RelativeRangePicker from './relative-range';
+import ModeSwitcher from './mode-switcher';
+import clsx from 'clsx';
+import InternalAlert from '../alert/internal';
+import LiveRegion from '../internal/components/live-region';
+import { useDateRangePicker } from './use-date-range-picker';
+
+export interface DateRangePickerDropdownProps
+  extends Pick<
+    Required<DateRangePickerProps>,
+    | 'locale'
+    | 'isDateEnabled'
+    | 'isValidRange'
+    | 'value'
+    | 'relativeOptions'
+    | 'i18nStrings'
+    | 'dateOnly'
+    | 'timeInputFormat'
+    | 'rangeSelectorMode'
+  > {
+  startOfWeek: DayIndex;
+  isSingleGrid: boolean;
+
+  ariaLabelledby?: string;
+  ariaDescribedby?: string;
+}
+
+export function DateRangePickerEmbeddedDropdown({
```

Comment:
We want to have it embedded in a dropdown, but it is not a dropdown or?

Reply:
Agreed, I can definitely update the name here to exclude the dropdown part.